tests/test_clients.py

```diff
@@ -632,3 +632,19 @@ async def test_crossref_retraction_status(stub_data_dir: Path) -> None:
 def test_reformat_name(name: str, expected: str) -> None:
     result = reformat_name(name)
     assert result == expected, f"Expected '{expected}', but got '{result}' for '{name}'"
+
+
+@pytest.mark.vcr
+@pytest.mark.asyncio
+async def test_arxiv_doi_is_used_when_available() -> None:
```

Comment:
Any chance we can merge this test with another test, with explanatory comment or docs? Mainly, we are getting a lot of tests that do mostly the same thing

Also, feel free to ignore this

Reply:
nah -- I need it to be a different subset of scrapers (i.e. only Crossref then s2), and I don't want to reconfigure all of our other tests to take custom subsets of scrapers atm.